editor/src/document/properties_panel_message_handler.rs

```diff
@@ -84,34 +85,46 @@ impl DAffine2Utils for DAffine2 {
 	}
 }
 
-#[derive(Debug, Clone, Default, Serialize, Deserialize)]
+#[derive(Debug, Clone, Serialize, Deserialize, Default)]
 pub struct PropertiesPanelMessageHandler {
-	active_path: Option<Vec<LayerId>>,
+	active_selection: Option<(Vec<LayerId>, TargetDocument)>,
 }
 
 impl PropertiesPanelMessageHandler {
 	fn matches_selected(&self, path: &[LayerId]) -> bool {
-		let last_active_path = self.active_path.as_ref().and_then(|v| v.last().copied());
+		let last_active_path = self.active_selection.as_ref().and_then(|(v, _)| v.last().copied());
```

Comment:
It could be. I am trying to determine if the modified layer matches that of the active one. When I initially made this system I realized I could check by just seeing if the last value in the path was the same.

Reply:
So should the variable be called `last_active_id` rather than `last_active_path`?